README.md

```diff
@@ -83,46 +83,43 @@ All releases on the same Major version will be guaranteed to have backward compa
 
 ## Usage
 
-OSV-Scanner collects a list of dependencies and versions that are used in your project, before matching this list against the OSV database via the [OSV.dev API](https://osv.dev#use-the-api). To build the list of dependencies, you can point OSV-Scanner at your project directory, or manually pass in the path to individual manifest files.
+OSV-scanner parses lockfiles, SBOMs, and git directories to determine your project's open source dependencies. These dependencies are matched against the OSV database via the [OSV.dev API](https://osv.dev#use-the-api) and known vulnerabilities are returned to you in the output. 
 
-### Scan a directory
+### General Use Case: Scanning a Directory
 
-Walks through a list of directories to find:
+```console
+osv-scanner -r /path/to/your/dir
+```
 
-- Lockfiles
-- SBOMs
-- git directories for the latest commit hash
+The above command will find lockfiles, SBOMs, and git directories in your target directory and use them to determine the dependencies to check against the OSV database. It will give you the most complete understanding of the known vulnerabilities in your project. 
 
-which is used to build the list of dependencies to be matched against OSV vulnerabilities.
+The recursive flag `-r` or `--recursive` will tell the scanner to search all subdirectories in addition to the specified directory. It can find additional lockfiles, dependencies, and vulnerabilities. If your project has deeply nested subdirectories, a recursive search may take a long time. 
 
-Can be configured to recursively walk through subdirectories with the `--recursive` / `-r` flag.
+Git directories are searched for the latest commit hash. Searching for git commit has is intended to work with projects that use git submodules or a similar mechanism where dependencies are checked out as real git repositories. 
 
-Searching for git commit hash is intended to work with projects that use
-git submodules or a similar mechanism where dependencies are checked out
-as real git repositories.
+### Specify SBOM
 
-#### Example
+If you want to see the vulnerabilities only from dependencies in your SBOM use the following command:
```

Comment:
I feel like "check" is more natural here:

```suggestion
If you want to check for known vulnerabilities only in dependencies in your SBOM use the following command:
```

Reply:
I agree about the word "check".  I think I prefer "from" to "in".